client/src/components/RelatedItems/RelatedItemsEntry.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+// import styled from 'styled-components';
+import PropTypes from 'prop-types';
+
+// const Container = styled.div`
+//   text-align: center;
+// `;
+
+function RelatedItemsEntry({ item }) {
+  return (
+    <div id="relatedItemsEntry">
+      <img
+        src="https://images.unsplash.com/photo-1501088430049-71c79fa3283e?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=668&q=80"
+        width="200"
+        height="300"
+        alt=""
```

Comment:
It would be good if we could add some alt text here. It helps us make the app accessible and it also helps with finding components during tests. Something like {`Thumbnail for related item ${item.name}`} might be a good option.